src/integrator.jl

```diff
@@ -494,7 +494,7 @@ function build_subintegrator_tree_with_cache(
 
     # TODO fix mixed device type problems we have to be smarter
     return subintegrator_tree, init_cache(f, alg;
-        uprev=uprevouter, u=uouter, alias_u=true,
+        uprev=uprevouter, u=uouter, alias_u=false,
```

Comment:
shouldn't this be `alias_u=alias_u0`?

Reply:
Yeah, I think we should we propagating `alias_u0` down and doing this. Sanity check @termi-official ?

Reply:
This one here is a bit trickier, as it is independent of how u0 is treated. https://github.com/SciML/OrdinaryDiffEqOperatorSplitting.jl/blob/650ca71e9ab04ca3b3b760787aaa5311e991b6da/src/solver.jl#L24-L25

Reply:
I see. actually, I think we should leave this as it was before because of:
https://github.com/SciML/OrdinaryDiffEqOperatorSplitting.jl/blob/main/src/integrator.jl#L90